roles/openshift_facts/defaults/main.yml

```diff
@@ -147,20 +149,66 @@ openshift_node_groups:
     labels:
       - 'node-role.kubernetes.io/master=true'
     edits: []
+  - name: node-config-master-crio
+    labels:
+      - 'node-role.kubernetes.io/master=true'
+      - "{{ openshift_crio_docker_gc_node_selector | lib_utils_oo_dict_to_keqv_list | join(',') }}"
+    edits: "{{ openshift_node_group_edits_crio }}"
   - name: node-config-infra
     labels:
       - 'node-role.kubernetes.io/infra=true'
     edits: []
+  - name: node-config-infra-crio
+    labels:
+      - 'node-role.kubernetes.io/infra=true'
+      - "{{ openshift_crio_docker_gc_node_selector | lib_utils_oo_dict_to_keqv_list | join(',') }}"
+    edits: "{{ openshift_node_group_edits_crio }}"
   - name: node-config-compute
     labels:
       - 'node-role.kubernetes.io/compute=true'
     edits: []
+  - name: node-config-compute-crio
+    labels:
+      - 'node-role.kubernetes.io/compute=true'
+      - "{{ openshift_crio_docker_gc_node_selector | lib_utils_oo_dict_to_keqv_list | join(',') }}"
+    edits: "{{ openshift_node_group_edits_crio }}"
   - name: node-config-master-infra
     labels:
-      - 'node-role.kubernetes.io/infra=true,node-role.kubernetes.io/master=true'
```

Comment:
As long as you've tested this I'm fine with it, at one point this only worked with a single list item containing comma separated list of labels.